scanner/src/funTest/assets/analyzer-result.yml

```diff
@@ -191,3 +191,5 @@ analyzer:
     has_issues: false
 scanner: null
 evaluator: null
+labels:
```

Comment:
The labels have been added to test whether new labels set by the scan step are correctly merged with already existing ones. I was hoping, this could therefore fall under the part of the commit message which is about introducing a new test.